src/main/java/command/ManageCommand.java

```diff
@@ -70,33 +71,35 @@ public String getTestData() {
      * @param storage The Storage where changes are saved.
      */
```

Comment:
Oops, I think you might have forgotten @return tag with the relevant return object?